resume.md

```diff
@@ -1,49 +1,64 @@
+
+**Backend Developer**: Ensemble Education | Remote *August 2020 - October 2020*
+* Implemented Google OAuth2 and Facebook's login using the Graph API.
+* Implemented a file system through a closure table in PostGresSql
+* Assisted in the development of the AI module using scikit-learn to calculate expectation values and build a linear regression model
+
+**Software Research Assistant**: UIUC advertising department   | Remote *July 2020 - August 2020*
+* Developed an AI chatbot with DialogFlow in Node.js with a user interface in HTML/CSS. Has a user base of 200 surveyors.
+* Used to  gauge user reactions to sponsored posts vs non sponsored posts. This is being used in an UIUC advertising study.
+* Implemented a recognition and recommendation system for analysis and stored interactions in a database.
+
+**Undergraduate Data Science Researcher**: Yun Research Lab | Remote... *May 2020 - Augusr 2020*
+* Implemented features in the Social Media Macroscope for data analytics using Pandas, Numpy and scipy.
+* Used to assist social media researchers who need to do data analysis and don't know how to code
+* Built an object detection programs in Python for image processing utilizing YOLOV3 and cleaned datasets
+
+
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**Reddit bot**: *Makes requests to rate my professor and UIUC course APi and posts comments on reddits with relevant course info* |https://tinyurl.com/yyd39fs3 *August 2020*
+* Developed Reddit Professor in order to help students plan out classes by getting the RateMyProfessor rating and student reviews of their professors. Professors are integral to learning and this helps students understand which course section to take/whether or not to take the class.
+* Had problems with picking out information from a reddit post but was able to utilize features of Reddit's PRAW API's comment endpoints to optimize it
```

Comment:
point should be solution oriented vs problem oriented.  "Resolved X by using Reddit's PRAW API for blah blah"